docs/installingNative.md

```diff
@@ -181,48 +193,76 @@ Most of these prerequisites can be installed using the package manager of your O
 
   You may need to restart the PostgreSQL server after changing the file above.
 
-- Run the test suite (Docker must be installed and running):
-
-  ```sh
-  make test
-  ```
+## Running
```

Comment:
Consider a separate "configuration" heading?

Reply:
Changed in da99c72.